### ISSUE TITLE
Missing method to return data from mssql

### ISSUE THREAD

**Issue by yshehab** [thelazurite-cell/autobahn-core#30]
@thelazurite-cell The `MssqlConnector.RunCommand` does not return data to the caller, only logs it. Is there a way to return the data from this method?

**Comment by thelazurite-cell** [comment]
Yes it should be available, I'll review this and, push an update up to fix this.
